lib/fluent/plugin/out_influxdb.rb

```diff
@@ -55,14 +55,24 @@ def configure(conf)
                                               time_precision: @time_precision,
                                               use_ssl: @use_ssl,
                                               verify_ssl: @verify_ssl
-    
+
+    $log.info "Connecting to database: #{@dbname}, host: #{@host}, port: #{@port}, username: #{@user}, password = #{@password}, use_ssl = #{@use_ssl}, verify_ssl = #{@verify_ssl}"
+
+    @influxdb = InfluxDB::Client.new @dbname, host: @host,
+                                     port: @port,
+                                     username: @user,
+                                     password: @password,
+                                     async: false,
+                                     time_precision: @time_precision,
+                                     use_ssl: @use_ssl,
+                                     verify_ssl: @verify_ssl
+
     existing_databases = @influxdb.list_databases.map { |x| x['name'] }
```

Comment:
If someone hits the connection related problem, I will consider the verification improvement.